convert2rhel/repo.py

```diff
@@ -68,3 +74,35 @@ def restore_yum_repos():
 
     if not repo_has_restored:
         loggerinst.info("No .repo files to rollback")
+
+
+def get_eus_repos_available():
+    """Detected if the current system is an EUS one.
+
+    :return: The return can be either the path to the eus repos, or None, meaning it is not a EUS system.
+    :rtype: str | None
+    """
+    hardcoded_reposdir = _get_hardcoded_repofiles_dir()
+    reposdir = None
+    if os.path.exists(hardcoded_reposdir) and system_info.has_internet_access:
+        reposdir = hardcoded_reposdir
+
+    return reposdir
```

Comment:
```suggestion
```
I totally don't understand the purpose of this function. Decision whether the RHEL minor version corresponding to the to-be-converted-OS minor version shouldn't rely on what directories we store/hardcode. Because we also hardcode CentOS Linux 8.5 repositories, but corresponding RHEL 8.5 is not an EUS release,.